### ISSUE TITLE
Bot not downloading entire Spotify playlist

### ISSUE THREAD

**Issue by keux00** [An0nimia/DeezloaderAn0n_bot#17]
I did paste a ~160 songs playlist and the bot stopped at song ~80 or so, sending an error message to contact the author

**Comment by An0nimia** [comment]
@keux00 What is the error,m

**Comment by keux00** [comment]
I think the bot doesnt download songs that arent available in Deezer, had a few songs and did check it. A possible patch for this could be the bot downloading that "problematic" songs from YouTube but dont know how hard would it be to implement

**Comment by An0nimia** [comment]
@keux00 Are you serious? Wow I didn't notice that what strange a library called deezloader which description says I download songs from Deezer doesn't download songs which aren't readable on Deezer. Who would ever expected something like this? You should check the other codes which I wrote.

**Comment by keux00** [comment]
You dont need to be that sarcastic, I was just describing an error I had on telegram's bot. I also told you how to solve some songs since they are from Spotify AND arent in Deezer. As this bot also recognizes Spotify links. No need to be that rude and also im not disrespecting your work by any means.

**Comment by An0nimia** [comment]
@keux00 Man just laugh you too, it is my humour and anyway I apologize, because first I thought that you was talking about deezloader repository and no Deezloader An0n_bot. As second is this a bug or something? Just send me a pull request for fix these bug and as third yes it recognises the Spotify link, but I specified that they are converted to Deezer link.
Sorry If I appeared rude I didn't want I was just in the mood for be sarcastic.
P.S. There is already YouTube integration, but the code is outdated and for this there are errors
